<launch>
	<arg name="model" default="$(find robomuse_description)/urdf/robomuse.xacro"/>
        <arg name="multi_robot_name" default=""/>
  	<!--<arg name="gui" default="true" />-->
  	<!--arg name="rvizconfig" default="$(find robomuse_description)/rviz/urdf.rviz" /-->
	<!--<param name="use_gui" value="$(arg gui)"/>-->
	<!-- Load robot model -->
	<param name="robot_description" command="$(find xacro)/xacro '$(find robomuse_description)/urdf/robomuse.xacro' --inorder"/>
	
	  <!--Serial Node-->
   <!--node pkg="rosserial_python" type="serial_node.py" name="serial_node" output="screen">
     <param name="port" value="/dev/ttyACM0"/>
     <param name="baud" value="57600"/>
     <param name="tf_prefix" value="$(arg multi_robot_name)"/>
   </node-->	
    
    

    <!-- Load controller config -->
    <rosparam command="load" file="$(find robomuse_hardware)/config/robomuse_common.yaml"/>
    <rosparam command="load" file="$(find robomuse_hardware)/config/joint_publish.yaml"/>
    <!--param command="load" file="$(find_robomuse_hardware/config/robomuse_dyn.yaml" /-->	
    <node name="controller_spawner"
          pkg="controller_manager"
          type="spawner"
          output="screen"
          
	  args="robomuse_diff
		robomuse"/>

    <param name="left_wheel_radius_multiplier" value="0.5"/>

    <!-- To view the robot in RViz-->	
    <!--node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" /-->
    
   
    <!--node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" /-->

    <node name="robomuse_hw" pkg="robomuse_hardware" type="robomuse_hw" output="screen"/>

    <node name="robomuse_pose_server" pkg="robomuse_hardware" type="robomuse_pose_server"/>

</launch>
